signed/verifiers.go

```diff
@@ -55,7 +58,16 @@ func (v Ed25519Verifier) Verify(key data.PublicKey, sig []byte, msg []byte) erro
 	copy(sigBytes[:], sig)
 
 	var keyBytes [ed25519.PublicKeySize]byte
-	copy(keyBytes[:], key.Public())
+	pub := key.Public()
+	if len(pub) != len(keyBytes) {
```

Comment:
same value. keyBytes and array initialized to be of length ed25519.PublicKeySize, not a slice. I just did it this way for consistency with the signature length check (which was carried over from flynn/go-tuf)